### PR TITLE
[4.0] Client Selector label

### PR DESCRIPTION
**Pull request by brianteeman**
In the following views there is at least one select input in the toolbar. None of them have labels. This PR ensures that each input has a valid label.

image

image

com_associations
com_languages&view=overrides
com_languages&view=installed
com_menus
com_templates&view=styles
com_templates&view=templates

### CODE REVIEW

**Comment by brianteeman** [comment]
Updated branch to retrigger drone

**Comment by Quy** [comment]
There are overlapping changes with PR #24813. Which one to review?

**Comment by brianteeman** [comment]
as always review each pr on their own. if one is merged then the other may need adjusting

File: administrator/components/com_languages/layouts/joomla/searchtools/default.php [hunk 1]
```diff
@@ -61,6 +62,7 @@
 		<?php $client = $data['view']->filterForm->getField('client_id'); ?>
 		<div class="js-stools-container-selector">
 			<div class="js-stools-field-selector js-stools-client">
+			<div class="sr-only"><?php echo $client->label; ?></div>
```
Comment (Quy) [inline administrator/components/com_languages/layouts/joomla/searchtools/default.php]:
```suggestion
				<div class="sr-only"><?php echo $client->label; ?></div>
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3bbeb9e407417440d0fb4f040e399054a9ca4c10<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24889">issues.joomla.org/tracker/joomla-cms/24889</a>.</sub>

**Comment by brianteeman** [comment]
conflicts resolved

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 6515dc4a15bf59a5b67da241f904d8773d88c4b9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24889">issues.joomla.org/tracker/joomla-cms/24889</a>.</sub>

**Comment by ghost** [comment]
@Quy can you please retest?

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 055d8d1c60873d849207c02095529c2e392c0210<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24889">issues.joomla.org/tracker/joomla-cms/24889</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24889">issues.joomla.org/tracker/joomla-cms/24889</a>.</sub>

**Comment by brianteeman** [comment]
can this please be merged. I keep going to fix the code and then rememner I already did it

**Comment by infograf768** [comment]
tks

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**